catalogd/cmd/catalogd/main.go

```diff
@@ -42,6 +42,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/klog/v2"
 	"k8s.io/klog/v2/textlogger"
+	"k8s.io/utils/ptr"
```

Comment:
would it make sense to try and avoid pulling in a new dependency just for a small helper that returns a pointer to value?

Reply:
This isn't a new dependency: https://github.com/operator-framework/operator-controller/blob/f055efcb494520a2cb5fb5625dc76b6f2284af6e/go.mod#L38

Reply:
that 5000k lines vendor tricked me :)